fast_curator/catalogues/common.py

```diff
@@ -8,6 +8,12 @@
 else:
     from urlparse import urlparse
 
+try:
+    uproot_numentries = uproot.numentries  # uproot3
+except AttributeError:
+    import uproot3
+    uproot_numentries = uproot3.numentries
+
```

Comment:
I feel like this could be solved more "pythonically" (?) by doing:
```suggestion
try:
    from uproot import numentries
except AttributeError:
    from uproot3 import numentries
```
And then just use `numentries` in the code.

Alternatively, what do you think about this pattern:
```
```suggestion
try:
    import uproot
except ImportError:
    import uproot3 as uproot
```
I can imagine the latter being more trouble because of the API changes.

Reply:
```python
from uproot import numentries
```
will only work for `uproot3` and since uproot4 is now the default &rarr; fallback to `uproot3` for just that function.

Ideally, you also want to use `uproot4` wherever possible so
```python
try:
    import uproot
except ImportError:
    import uproot3 as uproot
```
will not solve the issue.